docker/Dockerfile

```diff
@@ -41,20 +42,22 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
     curl \
     smartmontools \
     tzdata \
+    procps \
+    xz-utils \
```

Comment:
note about `xz-utils`: I'm aware of the vulnerability - the compromised version was removed from the Debian upstream, and is not present in this image.